### PR TITLE
Deprecate this repository

### PR DESCRIPTION
**Pull request by paulmey**
Since this is now moved into golang.org/x/crypto/pkcs12, let's force everyone to use that version...

### CODE REVIEW

**Comment by paulmey** [comment]
@ahmetalpbalkan is this the right way?

**Comment by ahmetalpbalkan** [comment]
Maybe we should just delete the repo.

**Comment by dgryski** [comment]
Having

```go
func init() {
        log.Println("Azure/go-pkcs12: This package has moved to golang.org/x/crypto/pkcs12")
}
```

or equivalent is a bit nicer.

**Comment by ahmetalpbalkan** [comment]
That sounds worse than a compile-time error to me. If the depending program was doing something sensitive with the output then it can potentially break users.

**Comment by nathany** [comment]
It's nice keeping a README around for a while. The [amz package](https://github.com/go-amz/amz/tree/master) forces a compile time error with:

```go
package error

func error() {
	`ERROR: the correct import path is gopkg.in/amz.v1 ... `
}
```

Though no code at all would be a compile time error as well.

**Comment by paulmey** [comment]
the `// import` comment generates a error on `go get` using Go 1.4 and up

**Comment by ahmetalpbalkan** [comment]
I think a compilation error @nathany suggested is better.

**Comment by paulmey** [comment]
@ahmetalpbalkan @nathany better?

**Comment by nathany** [comment]
LGTM

File: error.go [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package error
+
+func error() {
+	`ERROR: the correct import path is golang.org/x/crypto/pkcs12 ... `
```
Comment (ahmetalpbalkan) [inline error.go]: no need for `...` ?

Comment (paulmey) [inline error.go]: analogous to amz. Besides, I love '...' most of my text messages end in '...'

Comment (ahmetalpbalkan) [inline error.go]: looks like txting but I'll let that slide lol

**Pull request merged**